### PR TITLE
OCPBUGS-24314: Modified a note in configuring linux cgroup section

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.14 +

Issue: [OCPBUGS-24314](https://issues.redhat.com/browse/OCPBUGS-24314)

Link to docs preview: [Preview](https://71843--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-cgroups-2#nodes-clusters-cgroups-2_nodes-cluster-cgroups-2)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by shajmakh** [comment]
/retitle OCPBUGS-24314: Modified a note in configuring linux cgroup section
fix typo in the PR title to enable the bot to recognize the Jira bug and attach the PR link to the ticket.

File: modules/nodes-clusters-cgroups-2.adoc [hunk 1]
```diff
@@ -37,7 +37,7 @@ endif::nodes[]
 
 [NOTE]
 ====
-Currently, disabling CPU load balancing is not supported by cgroup v2. As a result, you might not get the desired behavior from performance profiles if you have cgroup v2 enabled. Enabling cgroup v2 is not recommended if you are using performance profiles.
+Currently, disabling CPU load balancing is not supported by cgroup v2. As a result, you might not get the desired behavior from performance profiles. Applying performance profiles might automatically set the cluster back to cgroup v1.
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/nodes-clusters-cgroups-2.adoc:40]: Acknowledged..

**Comment by shajmakh** [comment]
@subhtk thank you for the PR!
I understand this is a change that is needed in the 4.14 docs. Your PR is posted against main (which I assume is for 4.16?). 
Also to make the bot happy the versions of the bug and the branch should align, so you need to adjust target version of the bug to 4.14.z:
https://github.com/openshift/openshift-docs/pull/71843#issuecomment-1953701115

**Comment by subhtk** [comment]
Let me change it. Thanks for the reviews.

**Comment by mrniranjan** [comment]
The wording used here: "Applying performance profiles might automatically set the cluster back to cgroup v1. 
Member" ,  it's not that it might happen,  it will definitely move the system back to v1 .

**Comment by mrniranjan** [comment]
Documentation should make it clear  that Having Performance profile with cgroupv2 is not possible.

**Pull request closed without merging**